packages/core/models/TxModel.js

```diff
@@ -7,6 +7,8 @@ import BigNumber from 'bignumber.js'
 import moment from 'moment'
 import { FULL_DATE } from './constants'
 import { abstractModel } from './AbstractModel'
+import Immutable from "immutable";
```

Comment:
Please use single quotes.
Please remove trailing semi-colon.

The same for the line 11.